src/ast/ast.cpp

```diff
@@ -429,7 +430,7 @@ std::string opstr(Binop &binop)
     case bpftrace::Parser::token::BOR:   return "|";
     case bpftrace::Parser::token::BXOR:  return "^";
     default:
-      std::cerr << "unknown binary operator" << std::endl;
+      LOG(FATAL) << "unknown binary operator"; // will abort here
       abort();
```

Comment:
Assuming you left the abort() in b/c of the static analysis warning, I'd rather have a comment to silence static analysis warning (if it exists, which it really should)

Reply:
Yea, my intention was to silence the 'control may reach end of non-void function'.

> I'd rather have a comment to silence static analysis warning

Hmm, I'm not sure what you mean. Do you mean leaving a comment here to specify the intention, or is there actually a way to silence the warning?

Reply:
> Hmm, I'm not sure what you mean. Do you mean leaving a comment here to specify the intention, or is there actually a way to silence the warning?

https://lgtm.com/help/lgtm/alert-suppression#c-c++

Reply:
I assume it doesn't silence warnings that local compiler gives, right? Is it alright to have the compiler warnings? I mean Idk if it's somehow annoying to people.

Reply:
No compiler warnings either, please. There should be a way to silence that too